docs/infra/standaarden/haven/index.md

```diff
@@ -0,0 +1,92 @@
+
+### ✅ Haven is een levende standaard
+Haven is een standaard die door verschillende gemeentes en leveranciers van gemeentes in de praktijk wordt gebruikt. Veel Leveranciers geven aan dat ze blij zijn met de Haven-standaard omdat het ze duidelijkheid geeft bij de communicatie met gemeenten.
+
+## Haven-compliant != direct compleet soeverein
+
+Als organisatie is het goed om na te denken over digitale soevereiniteit. Hoe meer gebruik je maakt van componenten waar je geen controle over hebt, hoe minder wendbaar je bent. Door je Kubernetes clusters op basis van de Haven-standaard in te richten zet je een stap in de goede richting. Echter is het **geen keurmerk voor totale soevereiniteit**.
+
+De volgende casus illustreert een situatie waarin je Haven-compliant bent, maar toch afhankelijk bent van platform-specifieke functionaliteit:
+
+- Op je cluster wil je een container hebben waar je bestanden kan wegschrijven. Binnen Kubernetes is het dan raadzaam om te werken met een `StorageClass`.
+- Die StorageClass mag volgens de Haven-standaard prima geïmplementeerd zijn met een Azure driver voor Azure Blob Storage of een AWS S3 Bucket.
+- Op dat moment maak je zelf de keuze om een vendor-specifiek product te gebruiken, de Haven-standaard dwingt niet af dat je dit niet doet.
+
+:::info[Met Haven niet direct compleet soeverein]
+
+Als je Haven-compliant bent, betekent dit niet direct dat je volledig platform-agnostisch bezig bent. De standaard staat het namelijk nog steeds toe om bijvoorbeeld platform specifieke functionaliteiten te gebruiken zoals `Azure Blob Storage` of `AWS S3 Buckets` voor file storage.
+
+:::
+
+### [Haven+](./haven-plus): meer soevereiniteit door componenten als vervanging voor Azure/ AWS services
+
+Gelukkig biedt het Haventeam een oplossing voor het bovenstaande probleem: een suite van componenten die functionaliteiten zoals `monitoring`, `authenticatie`, `databases`, `certificaatbeheer` en `secret management` voor je regelen. Hierdoor hoef je geen AWS- of Azure-specifieke functionaliteit te gebruiken.
+
+## Hoe maak ik mijn applicatie geschikt voor een Haven Cluster?
+De voorwaarden voor een project om op een Haven Cluster te kunnen draaien zijn:
+
+- De opzet moet containerized zijn (bijvoorbeeld met Docker)
+- Het project moet Kubernetes manifests bevatten (bijvoorbeeld op basis van Helm)
+- Er moet een endpoint beschikbaar zijn voor metrics
+- De applicaties moeten kunnen schalen naar meerdere replica's
+
+Op dit moment is er nog geen tooling of validator die checkt of je applicatie voldoet aan de voorwaarden om te draaien op een Haven Cluster.
+
+## Haven Compliancy Checker
+Deze CLI Tool stelt je in staat om pro-actief je Kubernetes cluster Haven-compliant te houden.
+
+- [Meer info over Haven Compliancy Checker](./haven-compliancy-checker)
+
+![Screenshot FSC Policy Builder](./img/schermafbeelding-compliancy-checker.png)
+
+
+## Haven+
```

Comment:
Dit is wat dubbelop, misschien samenvoegen met kopje "Haven+: meer soevereiniteit door componenten als vervanging voor Azure/ AWS services"?

Reply:
Check. Heb deze paragraaf weggehaald en een card toegevoegd aan het einde van het artikel zodat mensen als nog Haven+ kunnen vinden mochten ze er overheen gelezen hebben.